src/core/Application.js

```diff
@@ -54,16 +54,21 @@ export default class Application
          */
         this.ticker = new Ticker();
 
-        this.ticker.add(() =>
-        {
-            this.renderer.render(this.stage);
-        });
+        this.ticker.add(this.render, this);
 
         // Start the rendering
         this.start();
     }
 
     /**
+     * Render the current stage.
+     */
+    render()
```

Comment:
I'm not sure how I feel about render being a public function, when part of the appeal of the Application class is to hide boiler plate stuff away from developers. Would they ever want to call render() manually when the ticker is already calling it when it can automatically?